v3/setup/klam-ssh.sh

```diff
@@ -0,0 +1,137 @@
+#!/bin/bash
+
+AZ=$(curl -s http://169.254.169.254/latest/meta-data/placement/availability-zone)
+REGION=${AZ::-1}
+ROLE_NAME="$(etcdctl get /klam-ssh/ROLE_NAME)"
+ENCRYPTION_ID="$(etcdctl get /klam-ssh/ENCRYPTION_ID)"
```

Comment:
OK, I fixed these up.  There will need to be an update to infrastructure as well so that the example configs have the values corrected.